repos/system_upgrade/el7toel8/actors/scanmemory/libraries/library.py

```diff
@@ -0,0 +1,22 @@
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+def _get_memoryinfo():
+    """Return lines from /proc/meminfo."""
+    # Expecting the file exists on earch system, skipping any check
+    try:
+        return dict((i.split()[0].rstrip(':'), int(i.split()[1])) for i in open('/proc/meminfo').readlines())
```

Comment:
Please use context manager with open. Also exception handling should be in place - what if data isn't parsed the way you expected? Traceback in the framework is not sexy

Reply:
fixed - btw, I removed the try as regarding the comment it didn't make sense - unless the comment is wrong.